src/main/blackbox/blackbox.c

```
@@ -104,14 +107,15 @@ void checkFlashStop(void);
 #define DEFAULT_BLACKBOX_DEVICE     BLACKBOX_DEVICE_NONE
 #endif
 
-PG_REGISTER_WITH_RESET_TEMPLATE(blackboxConfig_t, blackboxConfig, PG_BLACKBOX_CONFIG, 4);
+PG_REGISTER_WITH_RESET_TEMPLATE(blackboxConfig_t, blackboxConfig, PG_BLACKBOX_CONFIG, 5);
 
 PG_RESET_TEMPLATE(blackboxConfig_t, blackboxConfig,
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If PG_RESET_TEMPLATE is a macro then please configure it.

(unknownMacro)